pkg/config/containers.conf

```diff
@@ -143,7 +143,7 @@
 #
 # ipcns = "private"
 
-# container engines use container separation using MAC(SELinux) labeling.
+# container engine use container separation using MAC(SELinux) labeling.
```

Comment:
not sure I would have made this and the next change.

Reply:
Probably did not have to.  I think this is a mistake.  I originally globally changed libpod->engines and then changes engines->engine.